.gitignore

```diff
@@ -1,3 +1,4 @@
+*.swp
```

Comment:
That's editor-specific and IMHO belongs in your local `~/.gitignore` (same as `tags`, `*.swo` etc.)

Reply:
Huh. Funny enough I *have* `*.swp` in my `~/.gitignore` but git seems to ignore it.

Reply:
Reading https://git-scm.com/docs/gitignore it seems that the global ignore file resides in `$XDG_CONFIG_HOME/git/ignore`

Reply:
But fair enough.